static/app/views/replays/detail/layout/focusTabs.tsx

```diff
@@ -32,9 +36,12 @@ type Props = {
 function FocusTabs({isVideoReplay}: Props) {
   const organization = useOrganization();
   const {getActiveTab, setActiveTab} = useActiveReplayTab({isVideoReplay});
+  const hasLogsFeature =
+    organization.features.includes('ourlogs-enabled') &&
+    organization.features.includes('ourlogs-infinite-scroll');
```

Comment:
I don't see a ff for this, could you add the replay team so we can poke around?

Reply:
I've been using our overrides more recently rather than options automator, do you not see it?
![Screenshot 2025-06-02 at 2 19 01 PM](https://github.com/user-attachments/assets/eea25021-b277-4469-a22b-6672391f8c6b)